internal/views/restartable_resource.go

```diff
@@ -0,0 +1,52 @@
+package views
+
+import (
+	"errors"
+
+	"github.com/derailed/k9s/internal/resource"
+	"github.com/derailed/k9s/internal/ui"
+	"github.com/gdamore/tcell"
+)
+
+type (
+	restartableResourceView struct {
+		*resourceView
+	}
+)
+
+func newRestartableResourceViewForParent(parent *resourceView) *restartableResourceView {
+	v := restartableResourceView{
+		parent,
+	}
+	parent.extraActionsFn = v.extraActions
+	return &v
+}
+
+func (v *restartableResourceView) extraActions(aa ui.KeyActions) {
+	aa[tcell.KeyCtrlR] = ui.NewKeyAction("Restart Rollout", v.restartCmd, true)
```

Comment:
We already bind `ctrl+r` to refresh the resource list. 

Maybe, we could change it to another combination, maybe `ctrl+t`, since it is used to  `trigger` a CronJob and a Restart is _somewhat_ related?